pom.xml

```diff
@@ -20,7 +20,7 @@
     <artifactId>up-client-mqtt5-java</artifactId>
     <name>MQTT5 Transport Library for uProtocol with java</name>
     <description>MQTT5 Transport Library for uProtocol with java</description>
-    <version>0.1.0-SNAPSHOT</version>
+    <version>0.2.0-alpha-SNAPSHOT</version>
```

Comment:
You can call it 0.2.0, you don't need the alpha

Reply:
I want to mark it as alpha, because this is a implementation for up-spec 1.6.0-alpha.2. After the up-spec got released and is ready for using in production (at least this version) I will remove the prefix here too

Reply:
Alpha term is only needed for the spec, here you can continue to use 0.x.x

Reply:
as discussed with Kai, we will remove the -alpha suffix, since this is a 0-mayor version either. So the 0-mayor release is showing, that this is a "work in process" library anyway